### PR TITLE
Retire windows build script

### PR DESCRIPTION
**Pull request by gdong1**
Update build tool to use Buildloader.py to build SBL
in both Linux and Windows.
Update tools to support multiple workspaces, so platform
packages could be in another place.

TEST=Build qemu and APL platform success.

Change-Id: I2482037ba605218c947b6de28abe8e3eeacdc17f
Signed-off-by: Guo, Dong <guo.dong@intel.com>

### CODE REVIEW

File: BootloaderCorePkg/Tools/PrepareFspBin.py [hunk 1]
```diff
@@ -38,20 +38,18 @@ def CloneFspRepo (fsp_dir):
     print ('Done\n')
 
 
-def CheckFileListExist (copy_list, sbl_dir):
+def CheckFileListExist (copy_list):
```
[the remark this answers is not in the document]
Comment (gdong1) [reply BootloaderCorePkg/Tools/PrepareFspBin.py]: sure. will roll back the change

File: BootloaderCorePkg/Tools/PrepareFspBin.py [hunk 2]
```diff
@@ -38,20 +38,18 @@ def CloneFspRepo (fsp_dir):
...
     exists = True
     for src_path, dst_path in copy_list:
-        dst_path = os.path.join (sbl_dir, dst_path)
         if not os.path.exists(dst_path):
             exists = False
             break
     return exists
 
-def CopyFileList (copy_list, fsp_dir, sbl_dir):
+def CopyFileList (copy_list, fsp_repo_dir):
```
[the remark this answers is not in the document]
Comment (gdong1) [reply BootloaderCorePkg/Tools/PrepareFspBin.py]: sure. will roll back the change

File: BootloaderCorePkg/Tools/PrepareFspBin.py [hunk 3]
```diff
@@ -62,24 +60,25 @@ def CopyFileList (copy_list, fsp_dir, sbl_dir):
     print ('Done\n')
 
 
-def CopyFspBins (fsp_dir, sbl_dir, platform):
+def CopyFspBins (fsp_repo_dir, fsp_dir, platform, boardname):
```
[the remark this answers is not in the document]
Comment (gdong1) [reply BootloaderCorePkg/Tools/PrepareFspBin.py]: sure, will roll back the change

File: BootloaderCorePkg/Tools/PrepareFspBin.py [hunk 4]
```diff
@@ -163,28 +162,36 @@ def BuildFspBins (fsp_dir, sbl_dir, platform, flag):
...
-    if len(sys.argv) < 2:
+    if not os.path.exists(workspace_dir):
+        os.makedirs(workspace_dir)
+
+    if len(sys.argv) < 3:
         print ('Platform argument is required!')
         return -1
 
-    platform = sys.argv[1]
+    fspfile          = sys.argv[1]
```
[the remark this answers is not in the document]
Comment (gdong1) [reply BootloaderCorePkg/Tools/PrepareFspBin.py]: sure. will update the patch.

**Comment by mauricema** [comment]
It looks good to me now.

**Pull request merged**